.github/workflows/build-book-release.yml

```diff
@@ -157,7 +159,10 @@ jobs:
           ## For running the checks
           message(paste('****', Sys.time(), 'installing rcmdcheck and BiocCheck ****'))
           remotes::install_cran("rcmdcheck")
+          remotes::install_cran("MLeval")
           BiocManager::install("BiocCheck")
+          BiocManager::install("GUniFrac")
+          BiocManager::install("rgl") 
```

Comment:
You could check if we could move some or all of these into DESCRIPTION file Suggests/Depends/Imports?

Then they should be automatically included I think.

If that would cause trouble we can move those cases back here. This GHA file should left only for special cases.

Reply:
Moved some to DESCRIPTION file.